js/transform.js

```diff
@@ -119,8 +119,18 @@ function getNameAndTaxYear(inputDom) {
 function getListOfForms(inputDom) {
     var returnData = inputDom.getElementsByTagName('ReturnData')[0];
     var children = returnData ? Array.prototype.slice.call(returnData.children) : [];
+    var childCounts = {};
     return children.map(function(child) {
```

Comment:
[Dust] Optional suggestions for this line:
1. Since you access `child.nodeName` a lot, you could create a new variable at the beginning `const { nodeName } = child`.
2. Instead of a new variable, you can also destructure it from the function like so: `children.map(({ nodeName }) => { ... })`. Whatever you feel is more readable.
3. I think an inline function is convention within a `.map`, so `children.map(child => { ... })`. I don't have much reasoning here lol except I'm used to it and more readable, feel free to take or leave.

Reply:
Great comments on style - it's been a while since I've written Javascript, so these notes (and all the others) are helpful, relevant, and appreciated!